### DISCUSSION TITLE
Time out error when exporting data to SQL server from dax studio

### DISCUSSION THREAD

**Discussion by vnavashree** [DaxStudio/DaxStudio#1176] [Q&A]
I have been receiving an error when I try to export data from DAX studio to a SQL server. The error I'm getting is below.

![image](https://github.com/DaxStudio/DaxStudio/assets/143633791/14f7ad28-1654-4f14-84e1-7cabd7500aa7)

Does anyone know what I can do to resolve this?

**Comment by dgosbell** [comment]
Answered in #1175
